### ISSUE TITLE
Where maps are used as inputs, ensure accessible names are present

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-modules/Mapping#143]
Where map elements are used as inputs (admin and user settings, page blocks, etc.), corresponding labels should be applying. Currently, these elements are typically described by their grouped controls, and while controls should be accessible and properly labeled for interactions within the map element, maps are in need of their own labels to describe expected input (e.g. "Default bounds" for map elements in admin and user settings).

**Comment by jimsafley** [comment]
Fixed in 265d380bfb29ab0580c6ff6ab9fc407f08636b20
